docs/_includes/page-navigation.njk

```diff
@@ -22,15 +22,15 @@
 <hr class="major opacity-50 mb-0"/>
 <div class="container-fluid py-12">
   <div class="row justify-content-between">
-    <div class="col-6 col-md-3 col-lg-2">
+    <div class="col-6 col-md-3 col-lg-4 col-xl-3">
```

Comment:
Should this file be updated to match the previous/next action button markup?

Reply:
i could go either way on this. documentation may not need to emphasize going to the next screen. (but a sequential process might want to emphasize going to the next screen.) maybe leave this as is for now